src/algo/ClosedHash.js

```diff
@@ -136,6 +137,7 @@ export default class ClosedHash extends Hash {
 	}
 
 	resizeInitialTable() {
+		this.resetAll();
```

Comment:
Can you move this line below the line `this.commands = [];` (currently on 142)? 

Calling `resetAll()` causes some animation changes to occur, but in order for them to be properly implemented they have to be stored in the `this.commands` array which is returned to the original `implementAction` call. Having the lines in the order they're currently in causes some items not to be deleted (like in the video) 

https://github.com/RodrigoDLPontes/visualization-tool/assets/67525591/e7991b4f-9b76-4520-8da9-c92bbfae8925

Reply:
Basically we just want the commands array to be emptied before making any animation calls, but never again once the animation (`this.cmd`) calls begin.